_component_design/breadcrumbs.md

```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
 
 <section class="static-section" markdown="1">
 
-Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page hierarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second level of nesting and deeper within a module.
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use breadcrumbs when you want to improve web application way-finding and navigation. Breadcrumbs help the user understand their location within an application, and provide an easy way for the user to navigate out of their current view.
```

Comment:
@araiford 
Should we give guidance on when NOT to use breadcrumbs?

Reply:
Will ask Drew on 1/7/2019.

Reply:
Done.